.eslintrc.js

```diff
@@ -10,7 +10,8 @@ module.exports = {
     parser: '@typescript-eslint/parser',
     ignorePatterns: [
         '**/{node_modules,lib}',
-        '**/sprotty-local-template/**/*'
+        '**/sprotty-local-template/**/*',
+        '**/lib/**/src/**/*'
```

Comment:
Does the pattern above `'**/{node_modules,lib}'` fail to ignore the lib content?

Reply:
Yes, I needed to add `'**/lib/**/src/**/*'`

Reply:
As otherwise source folders in the new lib folder were considered.

There is room for optimization in the examples, but in another PR. 🙂